ppr-ui/src/components/common/DatePicker.vue

```diff
@@ -10,7 +10,7 @@
       </v-col>
       <v-col
         class="picker-title pl-4"
-        :class="{ 'picker-err': endDate === null && datePickerErr }"
+        :class="{ 'picker-err': endDate === null && datePickerErr && !disableEndDate }"
```

Comment:
Maybe we could rename the new prop to `enableEndDate` for clarity?

Reply:
Good catch.. This code is not required after the change to allow for date range..